app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentsListStates.kt

```diff
@@ -0,0 +1,33 @@
+package org.simple.clinic.home.overdue
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class OverdueListSectionStates(
+    val pendingListState: PendingListState,
+    val isPendingHeaderExpanded: Boolean,
+    val isAgreedToVisitHeaderExpanded: Boolean,
+    val isRemindToCallLaterHeaderExpanded: Boolean,
+    val isRemovedFromOverdueListHeaderExpanded: Boolean,
+    val isMoreThanAnOneYearOverdueHeader: Boolean
+) : Parcelable {
+
+  fun pendingListStateChanged(pendingListState: PendingListState): OverdueListSectionStates =
+      copy(pendingListState = pendingListState)
+
+  fun pendingChevronStateIsChanged(isPendingHeaderExpanded: Boolean): OverdueListSectionStates =
```

Comment:
nit: Since this is a state object. Let's remove chevron from the name and just call is `pendingHeaderStateChanged` or something like that. Similar to how we named `pendingListStateChanged`. WDYT?